build.sbt

```diff
@@ -342,25 +310,35 @@ lazy val aws = projectMatrix
  */
 lazy val `aws-http4s` = projectMatrix
   .in(file("modules/aws-http4s"))
-  .dependsOn(aws, complianceTests % "test->compile", dynamic % "test->compile")
+  .dependsOn(`aws-kernel`, `http4s-kernel`, json, xml, complianceTests % "test->compile", dynamic % "test->compile")
```

Comment:
**issue:** I don't think we actually use either `dynamic` or `complianceTests` here.